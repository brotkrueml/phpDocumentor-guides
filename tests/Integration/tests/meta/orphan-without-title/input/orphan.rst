:orphan:

A page without title that is marked as `orphan` does not cause warnings.
