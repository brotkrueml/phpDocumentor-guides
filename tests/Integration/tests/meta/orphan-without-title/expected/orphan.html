<!DOCTYPE html>
<html lang="en">
<head>
    <title></title>


</head>
<body>
    <p>A page without title that is marked as <code>orphan</code> does not cause warnings.</p>
</body>
</html>
